src/controllers/admin.js

```diff
@@ -0,0 +1,93 @@
+/**=================================================================
+ * ====   TITLE::     ADDING ADMIN FORGOT AND RESET PASSWORD    ====  
+ * ====   AUTHOR:: jimoh19 <jemohkunle2007@gmail.com>           ====
+ * ====   DATE::            30TH JUNE 2020                      ====
+ * =================================================================
+ */
+
+const Admin = require('../models/admin');
+const crypto = require('crypto');
+const CustomError = require('../utils/CustomError');
+const bcrypt = require('bcrypt');
+const { sendForgotPasswordMail } = require('../EmailFactory/index');
+
+exports.adminForgotPassword = async (req, res) => {
```

Comment:
The link isn't working: might be because of the cryptographic hashing.
http://localhost:5000/api/admin/auth/reset-password/%08%12%ef%bf%bd@%ef%bf%bd%ef%bf%bd%ea%82%91e%0E%0C%19%08%ef%bf%bd%ef%bf%bd%ef%bf%bd)=%7C%ef%bf%bdE%ef%bf%bd)%17o%ef%bf%bd%ef%bf%bd%1Bv%ef%bf%bd 👀
Use a random string maybe